lib/util/cache/package/index.ts

```diff
@@ -60,13 +61,37 @@ export async function init(config: AllConfig): Promise<void> {
       get: redisCache.get,
       set: redisCache.set,
     };
-  } else if (config.cacheDir) {
+    return;
+  }
+
+  // istanbul ignore if
+  if (config.cacheDir && config.useSqliteCache) {
```

Comment:
Tests for this file are almost useless, but to make it useful, we have to initialize `memCache` properly, which is surprisingly complicated. Because of that, I think ignoring is okay for now.

Reply:
I don't like the ignore, we should refactor the tests